src/public/modules/forms/base/directives/submit-form.directive.js

```diff
@@ -90,44 +88,29 @@ function submitFormDirective(
         $('head').append(meta)
       }
 
-      scope.formLogin = function (authType, rememberMe) {
-        if (authType === 'MyInfo') {
-          return $q
-            .when(MyInfoService.createRedirectURL(scope.form._id))
-            .then((response) => {
-              setReferrerToNull()
-              $window.location.href = response.redirectURL
-            })
-            .catch((error) => {
-              console.error(error)
-              Toastr.error(
-                'Sorry, there was a problem with your login. Please try again.',
-              )
-            })
-        }
+      scope.formLogin = function (authType, isPersistentLogin) {
         // Fire GA tracking event
-        if (rememberMe) {
-          GTag.persistentLoginUse(scope.form)
-        }
+        if (isPersistentLogin) GTag.persistentLoginUse(scope.form)
 
-        // redirect to SPCP log in page
-        let destination =
-          '/' + scope.form._id + (scope.form.isPreview ? '/preview' : '')
```

Comment:
I don't think this is a regression because we don't allow logins in preview mode anyway, but could you help double check that we're not losing any functionality here?

Reply:
yeap, checked staging behaviour with prod - we present users with a view of the logged in form so we aren't losing functionality. users also can't logout (all buttons have no effect besides edit form)